package.sh

```diff
@@ -28,11 +28,14 @@ docker run --rm \
   -v $(pwd):/cyberark \
   packager \
   /bin/bash -c """
-  # Create pkg folder that is the repository of the final artefacts
+  # Create pkg folder that is the repository of the final artifacts
   mkdir /pkg
-  # Copy the final artefacts to pkg
+  mkdir /pkg/bin
```

Comment:
We could skip the preceding `mkdir /pkg` and do:
```
    mkdir -p /pkg/bin
    mkdir -p /pkg/buildpack
```

Reply:
yeah, you're right